x-pack/filebeat/input/azureeventhub/config_test.go

```diff
@@ -55,3 +55,92 @@ func TestValidate(t *testing.T) {
 		)
 	})
 }
+
+func TestValidateConnectionStringV1(t *testing.T) {
+	t.Run("Connection string contains entity path", func(t *testing.T) {
+		config := defaultConfig()
+		config.ProcessorVersion = "v1"
+		config.ConnectionString = "Endpoint=sb://my-namespace.servicebus.windows.net/;SharedAccessKeyName=my-key;SharedAccessKey=my-secret;EntityPath=my-event-hub;"
+		config.EventHubName = "my-event-hub"
+		config.SAName = "teststorageaccount"
+		config.SAKey = "my-secret"
+		config.SAContainer = "filebeat-activitylogs-event_hub_00"
+
+		if err := config.Validate(); err != nil {
+			t.Fatalf("unexpected validation error: %v", err)
+		}
+
+		assert.NotNil(t, config.ConnectionStringProperties.EntityPath)
+		assert.Equal(t, config.EventHubName, *config.ConnectionStringProperties.EntityPath)
+	})
+
+	t.Run("Connection string does not contain entity path", func(t *testing.T) {
+		config := defaultConfig()
+		config.ProcessorVersion = "v1"
+		config.ConnectionString = "Endpoint=sb://my-namespace.servicebus.windows.net/;SharedAccessKeyName=my-key;SharedAccessKey=my-secret"
+		config.EventHubName = "my-event-hub"
+		config.SAName = "teststorageaccount"
+		config.SAKey = "my-secret"
+		config.SAContainer = "filebeat-activitylogs-event_hub_00"
+
+		if err := config.Validate(); err != nil {
+			t.Fatalf("unexpected validation error: %v", err)
+		}
```

Comment:
[NIT]
same, you could use `require` here

Reply:
Replaced with `require`.